rosapi/scripts/rosapi_node

```diff
@@ -277,12 +303,12 @@ class Rosapi(Node):
 def get_service_host(request):
```

Comment:
note the comment above this line indicates that this service isn't used, so I'm not sure what to make of it

Reply:
Probably best to comment out or delete the code if it's not working 🤷

Reply:
Agreed, although I don't actually know what this code does so was erring on the safe side.

I will make a separate PR to remove it.